hilgendorf_moveit_config/config/hilgendorf.srdf

```diff
@@ -21,11 +21,36 @@
     <group name="left_eef">
         <link name="left_gripper_target" />
     </group>
+    <group name="both_arms">
+        <group name="right_arm" />
+        <group name="left_arm" />
+    </group>
+    <group name="whole_body">
+        <group name="right_arm" />
+        <group name="left_arm" />
+        <group name="right_eef" />
+        <group name="left_eef" />
+    </group>
+    <!--GROUP STATES: Purpose: Define a named state for a particular group, in terms of joint values. This is useful to define states like 'folded arms'-->
+    <group_state name="home" group="whole_body">
+        <joint name="left_elbow_joint" value="0" />
+        <joint name="left_shoulder_lift_joint" value="0.7766" />
+        <joint name="left_shoulder_pan_joint" value="-0.6001" />
+        <joint name="left_wrist_1_joint" value="0" />
+        <joint name="left_wrist_2_joint" value="0" />
+        <joint name="left_wrist_3_joint" value="0" />
+        <joint name="right_elbow_joint" value="0" />
+        <joint name="right_shoulder_lift_joint" value="2.2944" />
+        <joint name="right_shoulder_pan_joint" value="-0.8471" />
+        <joint name="right_wrist_1_joint" value="-3.1415" />
+        <joint name="right_wrist_2_joint" value="0" />
+        <joint name="right_wrist_3_joint" value="0" />
+    </group_state>
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="right_gripper" parent_link="right_gripper_target" group="right_eef" />
     <end_effector name="left_gripper" parent_link="left_gripper_target" group="left_eef" />
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="fixed_base" type="fixed" parent_frame="world" child_link="base_link" />
+    <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="base_link" />
```

Comment:
This is the standard that I've seen from original PR2 SRDF that I have been using... and unfortunately I have some dumb code in moveit_setup_assistant that does not work (fails gracefully) if the vjoint is not named this